src/main/java/com/dynatrace/jenkins/dashboard/TABuildWrapper.java

```diff
@@ -73,8 +73,8 @@
 	/**
 	 * The 1st arg is system profile name, the 2nd is build number
 	 */
-	private static final String RECORD_SESSION_NAME = "%s_Jenkins_build_%s";
-
+	//private static final String RECORD_SESSION_NAME = "%s_Jenkins_build_%s";
```

Comment:
revert - treat that as a naming pattern, so you can change that to: `"%s_%s_Build-%s"`
update the javadoc